### ISSUE TITLE
Mission in dnav4 is hard to follow

### ISSUE THREAD

**Issue by ignaciofite** [CiscoDevNet/devnet-express-dna-issues#105]
While doing the [DNAc mission](https://github.com/CiscoDevNet/dnav3-code/blob/master/intro-dnac/06_mission/mission.py) yesterday in an event there were a bunch of issues.
- First of all the [solution](https://github.com/CiscoDevNet/dnav3-code/blob/solutions/intro-dnac/06_mission/mission_solution.py) doesn't work, it is broken or at least it was when I tried yesterday.
- Second, the use of recursive functions while it is a cool thing Python does I'm not very certain that it is the best approach for a DevNet Express where some networking guys have a hard time with python...

I attach [some code](https://github.com/CiscoDevNet/devnet-express-dna-issues/files/4514853/my_mission.txt) that was created by the attendees to make it work, maybe it helps. Even that code fails sometimes.

This mission should be rethought a little bit and made more resilient. Also, proctor-wise it would help a lot if the mission was able to send a webex message to the Mission Results room when attendees successfully complete it, it helps to track progress.

Thanks a lot guys! :D

**Comment by klevenstein** [comment]
Code has been updated and pushed to production: https://github.com/CiscoDevNet/dnav3-code/pull/86

Webex teams integration may be added as an enhancement at a later date.
